adb/fastboot.py

```diff
@@ -396,3 +406,17 @@ def Reboot(self, target_mode=b'', timeout_ms=None):
     def RebootBootloader(self, timeout_ms=None):
         """Reboots into the bootloader, usually equiv to Reboot('bootloader')."""
         return self._SimpleCommand(b'reboot-bootloader', timeout_ms=timeout_ms)
+
+    def Boot(self, source_file):
+        """
+        Fastboot boot image by sending image from local file system then issuing the boot command
+
+        :param source_file:
```

Comment:
Minor nit change this to the normal python args style

Reply:
Done